### PR TITLE
Release v0.8.3

### PR DESCRIPTION
**Pull request by leabeusch**
- [x] Fully documented, including `CHANGELOG.rst`

A small PR to have a commit that says v0.8.3.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -43,10 +61,13 @@ Internal Changes
 - Internal refactor: moved a number of inline comments to their own line (especially if
   this allows to have the code on one line instead of several) and other minor cleanups
   (`#98 <https://github.com/MESMER-group/mesmer/pull/98>`_).
+  By `Mathias Hauser <https://github.com/mathause>`_.
 - Refactor :py:func:`io.load_cmipng_tas` and :py:func:`io.load_cmipng_hfds` to
```
Comment (mathause) [inline CHANGELOG.rst:65]:
```suggestion
- Refactor ``io.load_cmipng_tas`` and ``io.load_cmipng_hfds`` to
```

**Review by mathause — APPROVED** [review]
The line length restrictions are not so strict on these files. You can change my `:py:func:` to double backticks - the cross reference was not working.

**Comment by leabeusch** [comment]
> The line length restrictions are not so strict on these files.

I know. But since we followed the same rules as in the code in most entries already anyways, I thought it wouldn't hurt to adapt the remaining "violations". ^^

**Comment by mathause** [comment]
I just realized - we should not have added the new section yet... Not a problem but not very nice either, sorry - my mistake.

**Pull request merged**